src/Arduboy2.h

```diff
@@ -430,6 +430,20 @@ class Arduboy2Base : public Arduboy2Core
    */
   void display(bool clear);
 
+  /** \brief
+   * Tells whether a given coordinate points to a valid pixel.
+   * 
+   * \param x The X coordinate of the pixel.
+   * \param y The Y coordinate of the pixel.
+   * 
+   * \details
+   * This function returns true if x is in the interval [0, WIDTH) and y is in
+   * the interval [0, HEIGHT).
+   */
+  bool validPixel(int16_t x, int16_t y) {
+    return !(x < 0 || x > (WIDTH-1) || y < 0 || y > (HEIGHT-1));
```

Comment:
The intent of putting the implementation right here is.. well, to give an inline hint.

Reply:
You'd be best off marking it as `static` and `constexpr` as well.

```cpp
static constexpr bool validPixel(int16_t x, int16_t y) {
    return !(x < 0 || x > (WIDTH-1) || y < 0 || y > (HEIGHT-1));
}
```